satpy/etc/enhancements/generic.yaml

```diff
@@ -930,3 +930,39 @@ enhancements:
           factor: 21.0
           min_stretch: 0.0
           max_stretch: 20.0
+
+  osisaf_sst_smhi_colormap:
```

Comment:
Is this the official osisaf colormap? in that case, `smhi` should be removed. Otherwise, this should not be here.

Reply:
Yes, as discussed above, there is not yet any official OSISAF SST coloring. Removed